tests/specs/npm/playwright_compat/main.ts

```diff
@@ -0,0 +1,4 @@
+import { chromium } from "npm:playwright";
+await chromium.launch();
+console.log("chromium launched");
```

Comment:
Does this actually work on CI?

Reply:
I suppose so. The first step in this spec test `run -A npm:playwright install chromium` should be downloading the chromium for this platform.

Reply:
If it's not cached already, it's worth looking into caching the chromium artifact.

Reply:
Added browser binary cache at key `playwright-${{ runner.os }}-${{ runner.arch }}`

Looks it's been saved successfully

<img width="1037" alt="Screenshot 2025-01-27 at 17 25 25" src="https://github.com/user-attachments/assets/7024e782-80ab-4967-b9f4-b4c326aa6d1e" />

https://github.com/denoland/deno/actions/runs/12984044432/job/36206262673

Reply:
Thanks for confirming